torch/futures/__init__.py

```diff
@@ -34,13 +34,7 @@ def __init__(self, *, devices: Optional[List[Union[int, str, torch.device]]] = N
         """
```

Comment:
I assume docs for `devices` arg will be added in follow PRs?

Reply:
I mentioned it in the paragraph but didn't add a specific section for it, let me do that.